index.html

```diff
@@ -4,12 +4,12 @@
 	<title></title>
 </head>
 <body>
-<div>
-    <input type="text" id="size" style="width: 40%" placeholder="How tall should the tree be?">
-</div>
-<div>
-    <input type="text" id="char" style="width: 40%" placeholder="Which character should be used?">
-</div>
+	<div>
+    	<input type="text" id="size" style="width: 40%" placeholder="How tall should the tree be?">
```

Comment:
(these inputs should actually be indented one line because they are a child of the div (because it wraps them)